### PR TITLE
New Artifactory and Quay Processors addded to downstream

### PR DESCRIPTION
**Pull request by gautambaghel**
I haven't done a dep ensure in this PR, lemme know and I can do it @msenmurugan

### CODE REVIEW

**Comment by gautambaghel** [comment]
no r2d2 here I guess :P

**Comment by gautambaghel** [comment]
Don't merge it, I have to do the cloud-auth thing too

File: examples/opssight.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "perceptor": {
       "name": "opssight-core",
       "port": 3001,
-      "image": "docker.io/blackducksoftware/opssight-core:2.2.3",
+      "image": "docker.io/blackducksoftware/opssight-core:2.2.5",
```
Comment (msenmurugan) [inline examples/opssight.json]: @gautambaghel It should be 2.2.4 right?

Comment (gautambaghel) [reply examples/opssight.json]: @msenmurugan - updated the examples

**Pull request merged**